types/index.ts

```diff
@@ -79,25 +27,25 @@ export enum QueryFilterComparisonType {
   ARRAY_INTERSECTS = 'arrayIntersects',
 }
 
-export interface ArrayIntersectsFilter {
+export type ArrayIntersectsFilter = {
```

Comment:
Just a Q: why change from interface to type?

Reply:
They are equivalent in that case. There I've changed so the extending pattern is consistent.
```typescript
export type AudienceQueryDefinition = {
  featureVersion: number;
  queryProperty: string;
} & AudienceDefinitionFilter;  // << matching here
```
As a rule of thumb, I try to use types for internal values or interface pieces and use interfaces for actual interfaces between business models or methods. Not a hard rule, just a convention I tend to use, so it is not really needed.